applications/incompressible_navier_stokes/flow_past_cylinder/application.h

```diff
@@ -143,7 +143,7 @@ class Application : public ApplicationBase<dim, Number>
     // clang-format off
     prm.enter_subsection("Application");
       prm.add_parameter("TestCase",     test_case,            "Number of test case.", dealii::Patterns::Integer(1,3));
-      prm.add_parameter("CylinderType", cylinder_type_string, "Type of cylinder.",    dealii::Patterns::Selection("circular|square"));
+      prm.add_parameter("CylinderType", cylinder_type, "Type of cylinder.");
       prm.add_parameter("CFL",          cfl_number,           "CFL number.",          dealii::Patterns::Double(0.0, 1.0e6), true);
```

Comment:
clang-format is off here, so optimally we would care manually about nice indentation here.

Reply:
I would like to go this way instead: https://github.com/exadg/exadg/pull/467 :)
Is this OK for you?

Reply:
I fixed this indentation in [7fd5759](https://github.com/exadg/exadg/pull/382/commits/7fd5759d839140a3d130e746455381be06bce9ed)